selftests/unit/test_archive.py

```diff
@@ -93,6 +93,37 @@ def test_tbz2_2_dir(self):
     def test_tbz2_2_file(self):
         self.compress_and_check_file('.tbz2')
 
+    def test_zip_symlinks(self):
+        """
+        Check that utils.archive can extract symlinks from zip file,
+        which is not supported on today's python's zipfile
+        """
+        def get_path(*args):
+            """ Get path with decompressdir prefix """
+            return os.path.join(self.decompressdir, *args)
+        zip_path = os.path.abspath(os.path.join(os.path.dirname(__file__),
+                                                "..", ".data",
```

Comment:
I noticed that we should be using `os.path.pardir` instead of `..`, but this can wait.  To not forget about it: https://trello.com/c/CwOBc8q2/734-use-symbolic-os-path-pardir-instead-of